extensions/iceberg/src/main/java/io/deephaven/iceberg/util/ColumnInstructions.java

```diff
@@ -0,0 +1,146 @@
+//
+// Copyright (c) 2016-2025 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.iceberg.util;
+
+import io.deephaven.annotations.BuildableStyle;
+import org.apache.iceberg.PartitionField;
+import org.apache.iceberg.PartitionSpec;
+import org.apache.iceberg.Schema;
+import org.apache.iceberg.types.Types.NestedField;
+import org.immutables.value.Value;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.OptionalInt;
+
+/**
+ * The instructions for mapping a Deephaven column in {@link Resolver#columnInstructions()}.
+ */
+@Value.Immutable
+@BuildableStyle
+public abstract class ColumnInstructions {
+
+    /**
+     * Create an unmapped column instructions, where neither schema field, nor partition field, is referenced.
```

Comment:
IntelliJ highlighted this

```suggestion
     * Create an unmapped column instructions, where neither schema field nor partition field is referenced.
```